packages/example-getting-started/src/repositories/todo.repository.ts

```diff
@@ -11,7 +11,7 @@ export class TodoRepository extends DefaultCrudRepository<
   Todo,
   typeof Todo.prototype.id
 > {
-  constructor(@inject('datasource') protected datasource: DataSourceType) {
+  constructor(@inject('datasources.ds') protected datasource: DataSourceType) {
```

Comment:
We also need to change the injection key to `datasources.db` if you were to apply @bajtos change